install/wavefront-operator/values.yaml

```diff
@@ -36,6 +36,11 @@ collector:
   ## Setting this to true is strongly recommended
   useDaemonset: true
 
+  ## If set to true, Collector pods will be upgraded automatically in case new minor upgrade version is available.
+  ## For pinning Collector to a specific version, you will need to set this option to false.
+  ## We support only minor version Auto Upgrades.
+  # enableAutoUpgrade: true
```

Comment:
While you're at it, can you please change the `collector.image.tag` to version `1.0.3`.